tests/basalprofileplugin.test.js

```diff
@@ -0,0 +1,82 @@
+var should = require('should');
+
+describe('basalprofile', function ( ) {
+
+  var basal = require('../lib/plugins/basalprofile')();
+
+  var sandbox = require('../lib/sandbox')();
+  var env = require('../env')();
+  var ctx = {};
+  ctx.data = require('../lib/data')(env, ctx);
+  ctx.notifications = require('../lib/notifications')(env, ctx);
+
+  function prepareSandbox ( ) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['prepareSandbox' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/771301346/issues/source?bid=2452379&fileBranchId=2479033#l13)